tests/conftest.py

```diff
@@ -3237,18 +3237,25 @@ def nb_ensure_endpoint_count(request):
             should_wait = True
 
     if should_wait:
-        # loggin the ready endpoint count for 5 min in 30sec internvals
-        for i in range(10):
-            ready_count = get_ready_noobaa_endpoint_count(namespace)
-            log.info(
-                f"Elapsed time {i * 30} sec: Waiting for noobaa endpoints to stabilize. "
-                f"Current ready count: {ready_count}"
+        # Wait for the NooBaa endpoint pods to stabilize
+        try:
+            for ready_nb_ep_count in TimeoutSampler(
+                30, 10, get_ready_noobaa_endpoint_count, namespace
```

Comment:
if I am not mistaken, `30, 10` is timeout and sleep respectively. This means that instead of 5 minutes timeout, you give it 30 seconds

Reply:
Right! Thanks. Fixed.